src/helper.ts

```diff
@@ -240,11 +240,11 @@ export async function handleStopInstallation(
     buttons: [t('box.stopInstall.keepInstalling'), t('box.yes'), t('box.no')],
   })
   if (response === 1) {
-    return sendKill(appName)
+    sendKill(appName)
   }
-  if (response === 2) {
+  else if (response === 2) {
+    sendKill(appName)
```

Comment:
The order here doesn't really makes difference because Javascript is async and these functions are both async.
On Electron, the send and on command are sync and the invoke/handle are async. so unless we change this on the main process to make the removeFolder to use a handle and here you change to invoke. So then the app would wait for the removal and then send the kill. But if the kill come first, even with these changes, it would make no diffrence since the remove folder is the last command 👍🏽

Reply:
This definitely fixed the issue, but I'll have to check.

Reply:
Since this doesn't functionally change anything and editing old commits is a massive pain, I'm just marking this as resolved.